metadata.json

```diff
@@ -34,7 +34,7 @@
   "requirements": [
     {"name":"puppet","version_requirement":">= 4.1.0"},
     {"name": "ruby","version_requirement": ">=2.2.3"},
-    {"name": "oneview-sdk-ruby","version_requirement": ">=5.6.0"}
+    {"name": "oneview-sdk-ruby","version_requirement": ">=5.9.0"}
```

Comment:
Is this version already released?

Reply:
this is the reason why build is failing..please replace it with 5.8.0

Reply:
need to release 5.9.0 from ruby then only this will work.

Reply:
we can't wait for the puppet PR's pending to be merged till ruby 5.9.0 is released. Moreover we can make this change in a new PR when puppet will be released.